index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title> Andy's Portfolio</title>
+    <script src="./scripts/script.js"></script>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta charset="UTF-8" />
+</head>
+
+<body>
+    <div class="slide-left">
+        <p>Andy Pierson's Portfolio</p>
+        
+    </div>
+<ul class="flex-container">
+    <a href="https://www.linkedin.com/in/andy-pierson-33368875/"><li class="flex-item">LinkedIn</li></a>
+    <a href="https://github.com/andyPiersonCodes"><li class="flex-item">GitHub</li></a>
+    <a href="./assets/WebDeveloperResume.PDF"><li class="flex-item">Resume</li></a>
+    <a href="aboutMe.html"><li class="flex-item">About Me</li></a>
+    <a href="#contactMe"><li class="flex-item">Contact</li></a>
+</ul>
+
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+
+
+<div id="contactMe" class="contact">
+    <h2>Contact Andy</h2>
+    <form>
+        <label for="fname"></label>
+        <input type="text" id="fname" name="firstname" placeholder="Your name..">
+
+        <label for="lname"></label>
+        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+        
+        
+        
+
+        <label for="reason"></label>
+        <select id="Reason" name="reason">
+            <option value="blank"></option>
+            <option value="interview">Interview Request</option>
+            <option value="freelance">Freelance Oppurtunity</option>
+            <option value="cool">You think I'm pretty cool</option>
+        </select>
+
+
+        
```

Comment:
_Nit_ Make sure youre not over using whitespace. Just 1 line between stuff if you feel necessary